formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/schema/Generation.kt

```diff
@@ -0,0 +1,413 @@
+@file:OptIn(ExperimentalSerializationApi::class)
+
+package kotlinx.serialization.protobuf.schema
+
+import kotlinx.serialization.ExperimentalSerializationApi
```

Comment:
General note: a single file with public methods that are not widely used will pollute the global scope, let's avoid that.

The easiest way to do so is to create an object `ProtobufSchemaGenerator` and declare all methods within this object

Reply:
We have only one public function here btw, so I doubt we need an object. 

Maybe move this method to a  `ProtoBuf` companion? Although it already has default methods for encoding/decoding....